File: common/src/main/java/dev/obscuria/fragmentum/registry/FragmentumRegistry.java
package dev.obscuria.fragmentum.registry;

import dev.obscuria.fragmentum.Fragmentum;

@SuppressWarnings("unused")
public interface FragmentumRegistry
{
    static Registrar registrar(String modId)
    {
        return Fragmentum.SERVICES.registrar(modId);
    }
}

File: common/src/main/kotlin/dev/obscuria/fragmentum/registry/FragmentumRegistry.kt
@file:Suppress("unused", "RedundantNullableReturnType")

package dev.obscuria.fragmentum.registry

import dev.obscuria.fragmentum.Fragmentum

object FragmentumRegistry {

    fun registrar(modId: String): Registrar {
        return Fragmentum.SERVICES.registrar(modId)
    }
}